pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -103,6 +104,12 @@ def get_first_recording_interface(converter):
             return interface
 
 
+def is_supported_recording_interface(recording_interface, metadata):
```

Comment:
```suggestion
def is_supported_recording_interface(recording_interface, metadata):
    """
    Temporary conditioned access to functionality still in development on NeuroConv.
    
    Used to determine display of ecephys metadata depending on the environment.
    
    Alpha build release should therefore always return False for this.
    """
```

Just adding a note to the docstring so we remember